src/module/Entity/src/Entity/Entity/Entity.php

```diff
@@ -253,6 +253,11 @@ public function isUnrevised()
         return $this->countUnrevised() > 0;
     }
 
+    public function isDead()
+    {
+        return !$this->getHead();
```

Comment:
HasCurrentRevisionCollectionFilter does not cover the case when the entity has no revision at all (Don't ask me how those are created anyway but there are some). This gives always a ugly error message (dead node something) on view and clutters the options for copying/moving.